todo-with-kuzzle-and-vueJS/README.fr.md

```diff
@@ -0,0 +1,724 @@
+# Todo MVC Step1
+## Pré-requis
+
+Node.js
+
+npm
+
+## Introduction
+Kuzzle permet de gérer de nombreuses données de manière très simple.
+
+Afin de l'installer, vous pouvez vous rendre [ici](https://docs.kuzzle.io/guide/1/essentials/installing-kuzzle/).
+
+Dans ce how-to, nous allons réaliser une simple todo-list utilisant 
+le pattern modèle-vue-controleur avec Kuzzle et VueJS.
+
+Cette première partie mettra en avant les fonctionnalités de 
+Kuzzle telles que:
+- la connexion à Kuzzle,
+- la création d'un index,
+- la vérification de l'existence d'un index,
+- la création d'une collection,
+- la récupération d'une liste des collections existantes,
+- la création d'un document,
+- la suppression d'un document,
+- la mise à jour d'un document,
+- la recherche de documents
+
+Les parties `<template>` et `style`, étant liées à VueJS et Vuetify mais
+n'ayant pas d'interaction avec Kuzzle, ne seront pas détaillées dans ce how-to;
+vous pouvez cependant consulter les fichiers concernés parallèlement à 
+la lecture de ce tutoriel.
+
+## Configuation du projet
+Commençons par installer Vue Cli : 
+```
+npm install -g @vue/cli
+```
+
+Nous pouvons maintenant créer un projet :
+```
+vue create todomvc
+```
+Ajoutez manuellement les fonctionnalités suivantes au projet : `router`
+et `vuex`.
+Sélectionnez ensuite '`Eslint + Standard config`', puis '`Lint on save`',
+et enfin '`In dedicated config files`'.
+Vous pouvez maintenant lancer votre projet via la commande :
+```
+npm run serve
+```
+puis vous rendre a l'adresse http://localhost:8080/.
+
+Nous allons maintenant mettre en place le store, qui nous permettra de
+stocker et d'utiliser certaines données devant être globales à notre 
+application.
+
+Pour cela, modifiez l'export du fichier `/src/store.js` de la façon suivante :
+```js
+export default new Vuex.Store({
+  state: {
+    connectedToKuzzle: false,
+    indexName: 'todolists'
+  },
+  mutations: {
+    setConnection(state, value) {
+      state.connectedToKuzzle = value;
+    }
+  }
+});
+```
+
+#### Framework et plugin
+Nous allons utiliser Vuetify pour les templates, 
+ainsi que vue-izitoast pour afficher des notifications.
+
+Afin de les installer lancez les commandes suivantes et
+choisissez la configuration par défaut :
+```
+vue add vuetify
+npm install vue-izitoast --save
+npm install material-design-icons-iconfont -D
+```
+Ajoutez ensuite les lignes suivantes dans le fichier `/src/main.js` :
+```js
+import VueIziToast from 'vue-izitoast';
+import 'material-design-icons-iconfont/dist/material-design-icons.css';
+import 'vuetify/dist/vuetify.min.css';
+import 'izitoast/dist/css/iziToast.css';
+
+Vue.use(VueIziToast);
+```
+
+## Se connecter à Kuzzle
+### Instanciation
```

Comment:
... I'm pretty sure I mentioned that one in my earlier reviews. 

Please check your text against a spell checker.

```suggestion
### Instantiation
```